draft-js-plugins-editor/src/Editor/index.js

```diff
@@ -24,14 +20,23 @@ export default class PluginEditor extends Component {
     plugins: React.PropTypes.array,
   };
 
+  static defaultProps = { plugins: [] };
+
   constructor(props) {
     super(props);
-    this.plugins = List(props.plugins).toArray();
-    const compositeDecorator = createCompositeDecorator(this.plugins, this.getEditorState, this.onChange);
+
+    // attach proxy methods like `focus` or `blur`
+    for (const method of proxies) {
+      this[method] = (...args) => (
+        this.refs.editor[method](...args)
```

Comment:
hehe, just added a return and figured: we are already returning the proxied return value because of the arrow function without curly braces